src/ripple/app/ledger/impl/LedgerConsensus.cpp

```diff
@@ -401,10 +401,10 @@ class LedgerConsensusImp
 
         // Adjust tracking for each peer that takes this position
         std::vector<NodeID> peers;
-        for (auto& it : mPeerPositions)
+        for (auto& i : mPeerPositions)
```

Comment:
When variables in a function are few, they should use "standard" names to avoid needless verbosity. If there were two or three loops running at once, then we might need a more descriptive name to distinguish them.